### PR TITLE
History (simple `history`)

### LINKED ISSUE #40: feat: simple history
- [x] `history`
- [x] `!` with number
- [x] `!!` for last command

### PR DESCRIPTION
**Pull request by RezaRahemtola**
# Description

History simple support with `history`.
Part of #40 

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have added the needed labels
- [x] I have linked this PR to an issue
- [x] I have linked this PR to a milestone
- [ ] I have linked this PR to a project
- [ ] I have tested this code
- [ ] I have added / updated tests (unit / functionals / end-to-end / ...)
- [ ] I have updated the README and other relevant documents (guides...)
- [ ] I have added sufficient documentation both in code, as well as in the READMEs

### CODE REVIEW

File: src/builtin/history.c [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+    free(history->command);
+    free(history->time);
+    free(history);
+}
+
+void builtin_history(env_t **env, char *const *args)
+{
+    (void)env;
+    (void)args;
```
Comment (Steci) [inline src/builtin/history.c:43]: No tea ? 🍵

**Review by Steci — APPROVED** [review]
Good Tea 🍵

File: src/builtin/history.c (hunk 1 above)
Comment (RezaRahemtola) [reply src/builtin/history.c:43]: Will implement error handling later

**Review by Croos3r — APPROVED** [review]
C'est bon mais je préfère le chocolat

**Pull request merged**